src/TaylorBot.Net/Program.Commands.Discord/src/TaylorBot.Net.Commands.Discord.Program/Modules/Taypoints/Domain/ITaypointBalanceRepository.cs

```diff
@@ -63,10 +63,26 @@ public async ValueTask<Result<ITaypointAmount, ParsingFailed>> ParseStringAsync(
                 return new RelativeTaypointAmount(4);
 
             default:
+                var inPercent = text.EndsWith("%");
```

Comment:
Please specify StringComparison and use the overload with `char` instead of `string`. See these for details
https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1310
https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1865-ca1867
```cs
text.EndsWith('%', StringComparison.OrdinalIgnoreCase)
```